test/transforms.jl

```diff
@@ -1356,6 +1356,40 @@
     @test Tables.isrowtable(n)
     rtₒ = revert(T, n, c)
     @test Tables.matrix(rt) ≈ Tables.matrix(rtₒ)
+
+    # ndim
+    x = rand(Normal(), 1500)
+    y = rand(Normal(), 1500)
+    z = rand(Normal(), 1500)
```

Comment:
Please create a more challenging test, these 3 variables are uncorrelated already even before the transformation. Create x = randn(1000) and then y = x + randn(1000)

Reply:
And z? x * y?